samples/GraphQL.DataLoader.Example/GraphQl/CarsBySalespersonDataLoader.cs

```diff
@@ -0,0 +1,25 @@
+using DataLoaderGql.Types;
+using GraphQL.DataLoader;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataLoaderGql.GraphQl;
+
+public class CarsBySalespersonDataLoader(DealershipDbContext db) : DataLoaderBase<int, List<Car>>
+{
+    protected override async Task FetchAsync(IEnumerable<DataLoaderPair<int, List<Car>>> list, CancellationToken cancellationToken)
+    {
+        var keys = list.Select(pair => pair.Key).ToHashSet();
```

Comment:
This `ToHashSet` would be helpful when used in the `Where` extension based on `IEnumerable`. But when it's based on `IQueariable`, you can leave it as non-enumerated `list.Select(pair => pair.Key)` without the risk of multiple enumerations. It would be enumerated only once to create the SQL `where in (...)` clause.